course4/homework1/test.cc

```diff
@@ -6,17 +6,17 @@
 
 int main()
 {
-	// to make sture the code is compiled in debug mode
+	// to make sure the code is compiled in debug mode
 #ifdef NDEBUG
 #error Compile the code in debug mode!
 #endif
 
-	std::cout << "Starting test..." << std::endl;	
+        std::cout << "Starting test..." << std::endl;
 
-	Vector2d a = {5, 6};
-	Vector2d b = {-1, 7};
+        Vector2d a = Vector2d::Rectangular(5, 6);
```

Comment:
You changed the indentation from tabs to spaces. 
It is OK to use any indentation scheme that suits you, but changing ti in existing code only creates "whitespace-only" changes in a patch, which increase noise-to-signal ratio.
Also, if this would be "real" pull request, I would probably reject it fro messing with my indents